src/window/advisor/military.c

```diff
@@ -231,6 +262,16 @@ static void button_empire_service(int legion_id, int param2)
     window_invalidate();
 }
 
+static void button_return_all_to_fort(int param1, int param2)
+{
+    int num_legions_not_at_fort = get_num_legions_not_at_fort();
+    if (num_legions_not_at_fort > 0) {
+        for (int i = 0; i < num_legions; i++) {
+            button_return_to_fort(i + 1, 1);
```

Comment:
Instead of calling `button_return_to_fort` (adding a weird `ignore_scrollbar` param), why not just create a `static void return_legion_to_fort(int legion_id)` that gets called directly by both `button_return_to_fort` and this function?